### ISSUE TITLE
valid WebDAV (DELETE, COPY or MOVE) requests with body must exit with 415

### ISSUE THREAD

**Issue by grgprarup** [owncloud/ocis#4332]
## Describe the bug
valid webDAV DELETE, COPY, or MOVE requests should not send a body in the request.
https://datatracker.ietf.org/doc/html/rfc4918#section-8.4

If a body is sent, then the server is supposed to return HTTP status 415 (Unsupported Media Type).

This has been implemented in reva PR https://github.com/cs3org/reva/pull/2974 for MOVE requests and might need implementation for COPY and DELETE requests.

## Steps to reproduce

1. Create user `uu1`
2. Create a file `test.txt`

```feature
curl -ks -uuu1:uu1 -XCOPY -H "DESTINATION: https://localhost:9200/remote.php/webdav/test2.txt" --data "doesnotmatter"  https://localhost:9200/remote.php/webdav/test.txt -v
```

```feature
curl -ks -uuu1:uu1 -XDELETE --data "doesnotmatter"  https://localhost:9200/remote.php/webdav/test.txt -v 
```

## Expected behavior
If a body is sent, then the server is supposed to return HTTP status 415 (Unsupported Media Type). 
COPY: `HTTP/1.1 415 Unsupported Media Type`
DELETE: `HTTP/1.1 415 Unsupported Media Type`

## Actual behavior
If a body is sent, then the body is ignored and the requests executed successfully.
COPY: `HTTP/1.1 201 Created`
DELETE: `HTTP/1.1 204 No Content`

**Comment by micbar** [comment]
fixed by #5131

**Comment by grgprarup** [comment]
The issue is fixed with PR https://github.com/owncloud/ocis/pull/5131. And verified with a curl request, which gives a response as expected.
```
curl -ks -uuu1:uu1 -XCOPY -H "DESTINATION: https://localhost:9200/remote.php/webdav/test2.txt" --data "doesnotmatter"  https://localhost:9200/remote.php/webdav/test.txt -v
```
`HTTP/1.1 415 Unsupported Media Type`
```
curl -ks -uuu1:uu1 -XDELETE --data "doesnotmatter"  https://localhost:9200/remote.php/webdav/test.txt -v 
```
`HTTP/1.1 415 Unsupported Media Type`
So closing this issue.
